app/router.cjsx

```diff
@@ -187,15 +189,17 @@ module.exports =
       </Route>
       <Route path="collaborators" component={require './pages/lab/collaborators'} />
       <Route path="media" component={EditMediaPage} />
-      <Route path="workflow/:workflowID" component={require './pages/lab/workflow'} />
+      <Route path="workflows/:workflowID" component={require './pages/lab/workflow'} />
       <Route path="workflow/:workflowID/visualize" component={require './pages/lab/workflow-viewer'} />
       <Route path="workflow/:workflowID/visualise" component={require './pages/lab/workflow-viewer'} />
```

Comment:
All of these workflow child routes can be nested into the new `workflows` route and be fixed to be `workflows/whatever-the-path-is` and redirects in place to make sure we aren't breaking anyone's book marks.